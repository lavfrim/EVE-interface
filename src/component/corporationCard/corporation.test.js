import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import CorporationCard from './index';

const corporation = {
    name: 'corp name',
    member_count: 143,
    description: 'corp description',
};

const ceoInfo = {
    name: 'ceo name',
};

describe('CorporationCard should render correctly', () => {
    it('with moch corporation ceoInfo', () => {
        const wrapper = mount(
            <CorporationCard corporation={corporation} ceoInfo={ceoInfo} />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

