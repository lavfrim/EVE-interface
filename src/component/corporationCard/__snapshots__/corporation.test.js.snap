// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CorporationCard should render correctly with moch corporation ceoInfo 1`] = `
<CorporationCard
  ceoInfo={
    Object {
      "name": "ceo name",
    }
  }
  corporation={
    Object {
      "description": "corp description",
      "member_count": 143,
      "name": "corp name",
    }
  }
>
  <p>
    Corporation name: corp name
  </p>
  <div
    className="corporation-card__ceo-link"
  >
    <p
      className="corporation-card__ceo-link-description"
    >
      CEO name:
    </p>
    <div
      className="corporation-card__ceo-link-link"
      onClick={[Function]}
    >
      ceo name
    </div>
  </div>
  <p>
    Member count: 143
  </p>
  <p>
    Description: corp description
  </p>
</CorporationCard>
`;
